openshift/openshift-ansible-contrib
# Set custom hostnames for servers

#### What does this PR do?
User can now set custom hostname prefix for roles (servers).

#### How should this be manually tested?
1) Remove `hosts` file in your inventory directory if it still exists.
2) In `<path to inventory>/group_vars/al.yml`, uncomment line(s) with variable for setting custom hostnames (`openstack_<role>_hostname`) and set their value to something else to your liking (but which is also a valid hostname).
3) Run the provisioning.
4) Display your servers with `openstack server list`. Note that servers with customised hostnames have prefixes according to variables in your inventory.

#### Is there a relevant Issue open for this?
--

#### Who would you like to review this?
cc: @tomassedovic @bogdando PTAL
